### ISSUE TITLE
TALLY.select(*).filter(DATE >= startDate)报错

### ISSUE THREAD

**Issue by fortitude1990** [stephencelis/SQLite.swift#890]
xcode10.2， filter内使用>=判断符，会导致      
``` 
 try connection.check(sqlite3_prepare_v2(connection.handle, SQL, -1, &handle, nil))

```
返回报错信息
near ":_": syntax error

**Comment by tanzolone** [comment]
Duplicate of #888

**Comment by lanephillips** [comment]
#888 is closed, but I've pod updated, and I'm still getting this error, too.

My failing line of code:

`try db.run(table.filter(timestamp < before).delete())`

(where "db" is a Connection, "table" is a Table, "timestamp" is a Expression\<Date>, and "before" is a Date)

This generates the statement `DELETE FROM "some_table" WHERE ("timestamp" <(_:_:) ?)`.

Apparently the Date gets encoded as `(_:_:) ?`

**Comment by lanephillips** [comment]
It turns out the podspec version has not been updated, so Cocoapods is downloading a stale version of the repo. I worked around this by linking my Podfile directly to the repo.

**Comment by Yigang0622** [comment]
You use like:
`let query = table.filter(0...10 ~= value)`
to filter out value between 0 and 10

**Comment by NathanFallet** [comment]
Should be fixed by 0.13.0. If not, please reopen.
